web/src/js/ducks/ui/tabs.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export enum Tab {
+    Capture,
+    FlowList,
+    Options,
+    Flow,
+}
+
+const tabsSlice = createSlice({
```

Comment:
This here is quite nice really! We should consider using it for more redux stuff going forward.

Reply:
oh that's really nice, so we shouldn't specify anymore stuff like this:
`const TOGGLE_MODE = "TOGGLE_MODE"`, right?

Reply:
yup, much nicer. The old style has benefits (it is more obvious what is actually happening), but I like it nonetheless!